package.json

```diff
@@ -12,33 +12,33 @@
     "test:all": "jest"
   },
   "dependencies": {
-    "@emotion/css": "^11.11.0",
-    "@emotion/react": "^11.11.0",
-    "@emotion/styled": "^11.11.0",
-    "@mui/icons-material": "^5.11.16",
-    "@mui/material": "^5.13.0",
-    "emoji-picker-react": "^4.4.9",
-    "react": "^18.2.0",
-    "react-dom": "^18.2.0",
-    "react-fast-marquee": "^1.6.2",
-    "react-hot-toast": "^2.4.1",
-    "react-router-dom": "^6.11.1",
-    "react-spring-bottom-sheet": "^3.4.1"
+    "@emotion/css": "11.11.0",
+    "@emotion/react": "11.11.0",
+    "@emotion/styled": "11.11.0",
+    "@mui/icons-material": "5.11.16",
+    "@mui/material": "5.13.0",
+    "emoji-picker-react": "4.4.9",
+    "react": "18.2.0",
+    "react-dom": "18.2.0",
+    "react-fast-marquee": "1.6.2",
+    "react-hot-toast": "2.4.1",
+    "react-router-dom": "6.11.1",
+    "react-spring-bottom-sheet": "3.4.1"
   },
   "devDependencies": {
-    "@types/react": "^18.0.28",
-    "@types/react-dom": "^18.0.11",
-    "@typescript-eslint/eslint-plugin": "^5.57.1",
-    "@typescript-eslint/parser": "^5.57.1",
-    "@vitejs/plugin-react": "^4.0.0",
-    "eslint": "^8.38.0",
-    "eslint-plugin-react-hooks": "^4.6.0",
-    "eslint-plugin-react-refresh": "^0.3.4",
-    "gh-pages": "^5.0.0",
-    "jest": "^29.7.0",
-    "ts-jest": "^29.1.1",
-    "typescript": "^5.0.2",
-    "vite": "^4.3.2",
-    "vite-plugin-pwa": "^0.15.1"
+    "@types/react": "18.0.28",
+    "@types/react-dom": "18.0.11",
+    "@typescript-eslint/eslint-plugin": "5.57.1",
+    "@typescript-eslint/parser": "5.57.1",
+    "@vitejs/plugin-react": "4.0.0",
+    "eslint": "8.38.0",
+    "eslint-plugin-react-hooks": "4.6.0",
+    "eslint-plugin-react-refresh": "0.3.4",
+    "gh-pages": "5.0.0", // Version kept as per original request
```

Comment:
@john-wayne630 Delete the comment. You cannot add comments in JSON format

Reply:
Done!

Reply:
Done!

Reply:
Done!

Reply:
Done!